sqlite3VdbeError(p, "cannot open savepoint - SQL statements in progress");